From 495fbe29d087383ab08cc49fba7be84bbc6530c0 Mon Sep 17 00:00:00 2001
From: Fugang Duan <b38611@freescale.com>
Date: Tue, 21 Apr 2015 19:29:16 +0800
Subject: [PATCH] MLK-10727-2 imx: clk-imx7d: remove imx7d enet 1588 clock
 parent init

Remove imx7d enet 1588 clock parent init from imx7d clock file since
1588 clock parent and rate is set in dts file.

Signed-off-by: Fugang Duan <B38611@freescale.com>
---
 arch/arm/mach-imx/clk-imx7d.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/arm/mach-imx/clk-imx7d.c b/arch/arm/mach-imx/clk-imx7d.c
index 5313bed..011949e 100644
--- a/arch/arm/mach-imx/clk-imx7d.c
+++ b/arch/arm/mach-imx/clk-imx7d.c
@@ -899,8 +899,6 @@ static void __init imx7d_clocks_init(struct device_node *ccm_node)
 	imx_clk_set_parent(clks[IMX7D_ENET_AXI_ROOT_SRC], clks[IMX7D_PLL_ENET_MAIN_250M_CLK]);
 	imx_clk_set_rate(clks[IMX7D_ENET_AXI_ROOT_CLK], 267000000);
 	imx_clk_set_parent(clks[IMX7D_ENET_PHY_REF_ROOT_SRC], clks[IMX7D_PLL_ENET_MAIN_25M_CLK]);
-	imx_clk_set_parent(clks[IMX7D_ENET1_TIME_ROOT_SRC], clks[IMX7D_PLL_ENET_MAIN_100M_CLK]);
-	imx_clk_set_parent(clks[IMX7D_ENET2_TIME_ROOT_SRC], clks[IMX7D_PLL_ENET_MAIN_100M_CLK]);
 
 	/* set uart module clock's parent clock source that must be great then 80Mhz */
 	if (uart_from_osc)
-- 
1.8.0

